mypy/checkexpr.py

```diff
@@ -3833,9 +3833,9 @@ def is_async_def(t: Type) -> bool:
     return isinstance(t, Instance) and t.type.fullname() == 'typing.Coroutine'
 
 
-def is_empty_tuple(t: Type) -> bool:
+def is_not_empty_tuple(t: Type) -> bool:
```

Comment:
I was a little confused by the name at first. What about renaming this to `is_non_empty_tuple`?

Reply:
Fixed.